test/sql/ordinality/ordinality_readcsv.test_slow

```diff
@@ -0,0 +1,18 @@
+# name: test/sql/ordinality/ordinality_readcsv.test_slow
+# description: Test read_csv function for multiple chunks
+# group: [ordinality]
+
+statement ok
+PRAGMA enable_verification
+
+
+query III
+SELECT * FROM read_csv('test/sql/ordinality/a1.csv') WITH ORDINALITY ORDER BY Numbers,Chars,ordinality;
+----
+7800 values hashing to d86e8a5e8df33cec28e0ed7b6c386419
```

Comment:
In general we prefer to avoid using the hash comparison feature - we can verify two results are the same using the labeled results, we can use that to verify results instead, e.g.:

```sql
query III nosort read_csv_result
SELECT * FROM read_csv('test/sql/ordinality/a1.csv') WITH ORDINALITY ORDER BY Numbers,Chars,ordinality;

query III nosort read_csv_result
SELECT *, row_number() OVER () AS ordinality FROM read_csv('test/sql/ordinality/a1.csv') ORDER BY Numbers,Chars,ordinality;
```